plugin.video.wizjatv/main.py

```diff
@@ -33,55 +36,38 @@ def WizjaTV():
     s, content = obj.ListaKanalow()
     content = json.loads(content)
     for item in content:
-        addDir(item['title'],item['url'] , 6, item['icon'],'','', False)
+        addDir(name=item['title'], url=item['url'], mode=PLAY_LINK, thumb=item['icon'], isFolder=False)
 
-def OdpalanieLinku():
+def OdpalanieLinku(url):
     try:
         global s
-        url = urllib.unquote_plus(params['url'])
         s, content = obj.ListaKanalow()
-        link = obj.Link(url,s)
+        link = obj.Link(url, s)
         xbmc.Player().play(str(link).replace("rtmp://$OPT:rtmp-raw=", ""))
     except:
         log_exception()
-        pass
 
 def get_params():
     paramstring = sys.argv[2]
     if paramstring.startswith('?'):
         paramstring = paramstring[1:]
-    return dict((k, vv[0]) for k, vv in parse_qs(paramstring).items())
+    return dict((k, urllib.unquote_plus(vv[0])) for k, vv in parse_qs(paramstring).items())
```

Comment:
Wcześniej `urllib.unquote_plus` było przy pobieraniu każdego elementu. Przeniosłem to własnie tutaj, żeby było raz.